blocks/url-input/index.js

```diff
@@ -109,9 +120,9 @@ class UrlInput extends Component {
 
 	onKeyDown( event ) {
 		const { selectedSuggestion, posts } = this.state;
-		// If the suggestions are not shown, we shouldn't handle the arrow keys
+		// If the suggestions are not shown or loading, we shouldn't handle the arrow keys
 		// We shouldn't preventDefault to allow block arrow keys navigation
-		if ( ! this.state.showSuggestions || ! this.state.posts.length ) {
+		if ( ! this.state.showSuggestions || ! this.state.posts.length || this.suggestionsRequest ) {
```

Comment:
If there had been a previous request which has since completed, wouldn't `this.suggestionsRequest` still be truthy because it's never unset?

Reply:
If there had been a previous request which has since completed, wouldn't `this.suggestionsRequest` still be truthy because it's never unset?

Good question. I don't know off the top of my head.

Reply:
Updated in 7ab4af6a2e356f2dfc6c802b7bffe29c0d213e83